com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Filter/DitherNode.cs

```diff
@@ -0,0 +1,55 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class DitherNode : IStandardNode
+    {
+        public static string Name = "Dither";
+        public static int Version = 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Version,
+            Name,
+@"
+{
+    uv = ScreenPosition.xy * _ScreenParams.xy;
+    uint index = (uint(uv.x) % 4) * 4 + uint(uv.y) % 4;
+    Out = In - DITHER_THRESHOLDS[index];
+}",
+            new ParameterDescriptor("In", TYPE.Vector, Usage.In),
+            new ParameterDescriptor("ScreenPosition", TYPE.Vec2, Usage.In),//TODO: ScreenPosition default support
```

Comment:
Is the idea that we'll go back and add in the "Default/Raw/Center/Tiled etc"  other screen position modes later? I'm not sure where we are at with proper support for dropdowns, I'm just curious

Reply:
Yeah I think screenposition default drop-down is one thing that we want to support later, I am not sure the status for the default input drop-downs but I think they were brought up by Ben a few weeks ago

Reply:
We don't have ANY referable data available right now - so the best we can do is make it a static vec2 and come back and add the real defaults later when they're available.  Tracy and I are keeping a list of these things that need to be added later.